frappe/desk/query_report.py

```diff
@@ -530,7 +530,7 @@ def get_data_for_custom_report(columns):
 
 
 @frappe.whitelist()
-def save_report(reference_report, report_name, columns):
+def save_report(reference_report, report_name, report_json):
```

Comment:
Wondering if there is a necessity to introduce a breaking change to this API 😅

Reply:
In the current scenario, the filters are not being saved to the report doc. So I think It's necessary to pass the filters too, with the columns.